app/components/common/pageHeader/pageHeaderDirective.js

```diff
@@ -0,0 +1,27 @@
+(function() {
+  'use strict';
+  
+  var template = require('./pageHeader.html');
+
+  module.exports = function pageHeader() {
+    var directive = {
+      restrict: 'EA',
+      scope: {},
+      controllerAs: 'vm',
+      bindToController: {
+        pageTitle: '@title',
+        pageSubtitle: '@subTitle',
+        nextElectionDate: '@nextElection'
+      },
+      link: link,
+      template: template,
+      controller: function() {} //if access to the controller is needed, add the controller in separate file
+    };
+    return directive;
+
+    function link(scope, el, attrs, vm) {
+      el.addClass('cfa-pageHeader');
```

Comment:
bc I forgot to commit the change where I added 'cfa-' to the 'pageHeader' class :open_mouth: 

It's updated now.